sklearn/feature_extraction/tests/test_image.py

```diff
@@ -13,7 +14,10 @@
     img_to_graph, grid_to_graph, extract_patches_2d,
     reconstruct_from_patches_2d, PatchExtractor, extract_patches)
 from sklearn.utils.graph import connected_components
+from sklearn.utils.testing import SkipTest
 
+if sp.__version__ < '0.12.0':
```

Comment:
@ogrisel I believe the second one wouldn't work as well because some versions of scipy have face in scipy.face() vs scipy.misc.face().